packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -100,9 +137,55 @@ export class LitDevTutorial extends LitElement {
   }
 
   /**
-   * Whether the tutorial is currently in its "solved" state.
+   * Whether the user has requested the solved code by clicking the solve button
```

Comment:
Maybe clarify this jsdoc a little? Wondering if `_requestSolvedCode` is going to get reset after the snackbar displays.